demosymfonyform/README.md

```diff
@@ -0,0 +1,26 @@
+# Demonstration of how to use PrestaShop Symfony form types
+
+## About
+
+This module demonstrates how to use existing PrestaShop Symfony form types inside a new page.
+
+ ### Supported PrestaShop versions
+
+ This module is compatible with and 1.7.8.0 and above versions.
+ 
+ ### Requirements
+ 
+  1. Composer, see [Composer](https://getcomposer.org/) to learn more
+ 
+ ### How to install
+ 
+  1. Download or clone module into `modules` directory of your PrestaShop installation
+  2. Rename the directory to make sure that module directory is named `demosymfonyform`*
+  3. `cd` into module's directory and run following commands:
+      - `composer install` - to download dependencies into vendor folder
+      - `composer dumpautoload` to generate autoloader for module
```

Comment:
I don't think this step is needed? 🤔

Reply:
I think it's better for use in production 🤔 indeed it might be useless here.

Reply:
In production you'll probably want to add options like `--optimize --no-dev` otherwise it's still useless 😅